crates/fluvio-cli-common/Cargo.toml

```diff
@@ -26,9 +26,14 @@ thiserror = "1.0.20"
 http = { version = "0.2", default-features = false}
 isahc = { version = "1.7", default-features = false, features = ["static-curl"] }
 futures = { version = "0.3" , default-features = false, features = ["std", "io-compat"]}
+tempfile = "3.3.0"
```

Comment:
this seems to be used only by the test, so should it be moved to `dev-dependencies`?